packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/FlexDemo/FlexDemo.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Flex, FlexItem } from '@patternfly/react-core';
+
+export class FlexDemo extends React.Component {
+  render() {
+    return (
+      <Flex>
+        <FlexItem>Flex item</FlexItem>
```

Comment:
can we add an example using the breakpointMods prop?

Reply:
Pretty much any of the lower variations use it. (Inline, flex-1, etc.)